### DISCUSSION TITLE
How do you use org-journal?

### DISCUSSION THREAD

**Discussion by casch-at** [bastibe/org-journal#435] [General]
At work I use it for quick note taking in meetings and to write down thoughts while working on a project.  It helps me a lot to keep my focus on the project/meeting.  I also use it for "free writing" (sich-leer-schreiben) when there is to much going on in my mind. I don't use encrypted journals at work.

At home I use it pretty much for "free writing" (sich-leer-schreiben) only, I do this mostly before I go to bad to empty my mind. I use encrypted journals at home.

I'm not sure if "free writing" is the same as "sich-leer-schreiben" in German, anyone? But I think the purpose is the same.

**Comment by bastibe** [comment]
I use it for meeting notes, and as a work journal, where I write down what I did in a day, what remains to be done, and reflect on how that went or could be improved. This is sort of an informal todo-list, without actually using org's TODO feature, org org-journal's TODO carryover.

Basically, my thinking aid.

Also, I have on occasion used org-journal from other editors than Emacs, and have written minimal implementations of goto-today, goto-yesterday/tomorrow, and search in other editors.
